master/static/migrations/20200401000000_initial.up.sql

```diff
@@ -402,7 +402,7 @@ CREATE TABLE public.templates (
 --
 
 CREATE TABLE public.trial_logs (
-    id integer NOT NULL,
+    id int8 NOT NULL,
```

Comment:
Hmm, can we get away without changing this? For someone creating a new database, running the migration will be essentially free anyway. Unless the database is somehow left in a dirtier state if we create and then update the column? I know we mentioned changing the initial up migration at the sync, but I can't convince myself at the moment that there's any point to it. It may not be _that_ likely to cause issues either, but allowing there to be more than one possible path through the migrations feels iffy.

Reply:
Yeah I agree. Will fix.